helmfile/02-gatekeeper.yaml

```diff
@@ -27,6 +27,7 @@ releases:
   labels:
     app: gatekeeper
   chart: upstream/gatekeeper
+  disableValidationOnInstall: true
```

Comment:
Maybe worth adding a comment with the rational here? It would be nice to keep track of if this was fixed upstream at some point so that you could still keep validation with CRD-installing charts.